eng/pipelines/runtime.yml

```diff
@@ -792,12 +792,28 @@ jobs:
     - FreeBSD_x64
     jobParameters:
       testScope: innerloop
-      testBuildPlatforms:
-      - Linux_x64
-      - windows_x64
-      - OSX_x64
       liveRuntimeBuildConfig: release
 
+#
+# Libraries Build that only run when coreclr is changed
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/build-job.yml
+    buildConfig: Debug
+    platforms:
+      - Linux_arm
+      - Linux_musl_arm
+      - Linux_musl_arm64
+    jobParameters:
+      liveRuntimeBuildConfig: release
+      # Don't run on isFullMatrix as we need the Release libraries build then,
+      # which is already handled above.
+      condition: >-
+        and(
+          eq(dependencies.evaluate_paths.outputs['SetPathVars_coreclr.containsChange'], true),
+          eq(variables['isFullMatrix'], false))
```

Comment:
Rather than making the IsFullMatrix condition a runtime condition, could you condition the whole job with a compile condition? That way the number of legs shown will be less on the rolling builds.